src/utils/hooks/useTheme.tsx

```diff
@@ -57,5 +57,6 @@ const color =
       return undefined
     }
 
-    return (theme.colors as { [key: string]: string })[colorName as Color]
+    // return the theme color or the string that was passed as a default (e.g., to support `white`).
+    return (theme.colors as { [key: string]: string })[colorName as Color] ?? colorName
```

Comment:
You are right, separating the returns is a good idea. I was just confused about the last line that didn't use the themeColor stored in the const. What do you think about this version?

```suggestion
    const themeColor = (theme.colors as { [key: string]: string })[colorName as Color]
    
    // return the color that was passed if it's not supported by the theme (e.g., for a theme-independent `white`).
    if(!themeColor) {
      return colorName
    }

    return themeColor
```

Reply:
oups, i missed your comment